react/react.d.ts

```diff
@@ -2105,6 +2105,7 @@ declare namespace __React {
         stroke?: string;
         strokeDasharray?: string;
         strokeLinecap?: string;
+        strokeLinejoin?: string;
```

Comment:
did you consider using string literal types?

```ts
strokeLinejoin?: "miter" | "round" | "bevel" | "inherit";
```

(from https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linejoin)